src/core/comparable.mli

```diff
@@ -5,6 +5,5 @@ module type S = sig
 
   include Equatable.S with type t := t
 
-  val compare:t Comparator.t
-end
-
+  val compare: t Comparator.t
```

Comment:
You might very well conflicts with other people file names. It's much safer (and consisten) to start all files with the same prefix. If a user once to change the name in its program he can simply use module aliases.